src/components/accession2/edit/CalculatorModal.tsx

```diff
@@ -154,13 +154,15 @@ export default function CalculatorModal(props: CalculatorModalProps): JSX.Elemen
           </Grid>
           <Grid item xs={12} textAlign='left'>
             <Box display='flex' textAlign='left' alignItems='end'>
-              <Textfield
-                label={strings.TOTAL_WEIGHT}
-                id='remainingQuantity'
-                onChange={(value) => onChangeRemainingQuantity(Number(value))}
-                type='text'
-                value={record.remainingQuantity?.quantity}
-              />
+              <Box width='600px'>
```

Comment:
@constanzauanini  this doesn't work in mobile view, extends beyond the screen.
I agree with @chudo76 we should handle this at the button level with a minimum height.. not make the modal bigger.
Also yeah if Get Count solves this in the near term (even with spanish).. we can certainly revisit this later.. but min-height would be a one-time fix that works for all..

Reply:
I will change the text for this button, and then will update the button component to have a min-height instead of height

Reply:
is that a web-components change or just in this one place?

Reply:
Height is set in web-components, for each button size (small, medium, etc)